charts/postgres-db-copy/ci/test-values.yaml

```
@@ -0,0 +1,19 @@
+podAnnotations:
+  logging: logging-splunk
+
+sourceDB:
+  host: "my-release-3-postgresql"
+  name: "postgres"
+  user: "postgres"
+  secret: "my-release-3-postgresql"
+  passwordKey: postgresql-password
+
+targetDB:
+  host: "my-release-3-postgresql"
+  name: "test"
+  user: "postgres"
+  secret: "my-release-3-postgresql"
+  passwordKey: postgresql-password
+
+tables:
+  - customers
```

```suggestion
podAnnotations:
  logging: logging-splunk

sourceDB:
  host: "my-release-3-postgresql"
  name: "sourceDB"
  user: "postgres"
  secret: "my-release-3-postgresql"
  passwordKey: postgresql-password

targetDB:
  host: "my-release-3-postgresql"
  name: "targetDB"
  user: "postgres"
  secret: "my-release-3-postgresql"
  passwordKey: postgresql-password

tables:
  - customers
```

Even though these are test values they should still be a more demonstrative example, the table names are very random right now